pkg/knowledge_base2/properties/bool_property_test.go

```diff
@@ -297,7 +297,8 @@ func Test_BoolPropertyValidate(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			assert := assert.New(t)
-			err := tt.property.Validate(tt.value, construct.Properties{})
+			resource := &construct.Resource{}
+			err := tt.property.Validate(resource, tt.value)
```

Comment:
nit: since resource isn't used for bool validate, can just simply `tt.property.Validate(nil, tt.value)`